### PR TITLE
OutputType based on TargetFramework

### PR DESCRIPTION
**Pull request by Faizan2304**
Issue: TestPlatform side fix for issue https://devdiv.visualstudio.com/DevDiv/_workitems?id=375688&_a=edit

Fix: To fix this issue we need to fix in two places
1) CPS side fix, Which already has been done. 
2) Fix in Microsoft.Net.Test nuget package to conclude OutputType type based on TargetFramework.  We cant do it in props file because that get included before we conclude TargetFramework, hence adding that logic in target file and deleting props file as it is not needed now.

Test:
Manually validated the fix.

### CODE REVIEW

**Comment by Faizan2304** [comment]
@srivatsn : Can you please point me the bug which CPS is tracking for this issue. As per my understanding this fix wont be needed once CPS will fix it completely.

File: scripts/build.ps1 [hunk 1]
```diff
@@ -328,9 +328,6 @@ function Create-NugetPackages
         Copy-Item $tpSrcDir\$file $stagingDir -Force
     }
 
-    # Copy and rename props file.
-    Copy-Item $tpSrcDir\"Microsoft.Net.Test.Sdk_props" $stagingDir\"Microsoft.Net.Test.Sdk.props" -Force
```
Comment (harshjain2) [inline scripts/build.ps1]: Comment out as we are going to add it again in future.

**Review by singhsarab — APPROVED** [review]

File: src/Microsoft.NET.Test.Sdk_props [hunk 2]
```diff
@@ -1,18 +0,0 @@
...
-          created a backup copy.  Incorrect changes to this file will make it
-          impossible to load or build your projects from the command-line or the IDE.
-
-Copyright (c) .NET Foundation. All rights reserved. 
-***********************************************************************************************
--->
-
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <!-- Output type for dotnet core test project should be exe. For full CLR it is Library by default. -->
```
Comment (harshjain2) [inline src/Microsoft.NET.Test.Sdk_props]: Comment out with appropriate bug url because of which we doing this temporary fix.

**Review by harshjain2 — APPROVED** [review]
Approved with few suggestions.

**Review by codito — APPROVED** [review]

**Review by smadala — CHANGES_REQUESTED** [review]
Moved these changes to #440. Please ignore this PR.

**Comment by srivatsn** [comment]
@Faizan2304 - https://github.com/dotnet/roslyn-project-system/issues/268 is the issue tracking reloading when targets change.

**Pull request merged**